projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/README.md

```diff
@@ -98,14 +96,25 @@ Note: These commands should be ran from `/jetpack-mu-wpcom` root directory.
+
+In most cases you will only need to sync the code to your sandbox, since Verbum is loaded through and on a simple site in all scenarios. There may be a case where you want to confirm that your PHP changes are not negatively affecting WoA sites (although Verbum will only load on Simple Sites, it doesn't hurt to be safe with PHP specifc code ). In this case, you will want to sync the code directly to your WoA using the steps above. 
 
 ### Local Development:
 
 To test your changes, you will first need to build the Verbum code using one of the build commands above and sync the changes to your development environment. The build commands will clean the `jetpack-mu-wpcom/src/build` directory and output the newly bundled files (`/build/verbum-comments`). One method to quickly sync and test changes is to use the `jetpack rsync mu-wpcom-plugin` command. After syncing the files, make sure that you sandbox your testing site to see your latest changes.
 
+Reminder:
+
+On Simple Sites: Sync the Verbum code changes to your sandbox and sandbox the site your testing.
+On Atomic Sites: Sync the Verbum code changes to your sandbox and sandbox `jetpack.wordpress.com`.
+
 ### After Merge - Deploy process
 
 1. Create a Jetpack PR, review it, and merge.
 2. To initiate a Simple Site deployment, follow these instructions: [https://fieldguide.automattic.com/developing-jetpack/jetpack-mu-wpcom/#simple-deployment]. The Jetpack release team (#jetpack-release) will also do a daily deployment for any merged changes, if you do not initiate a manual deployment.
-3. To initate a WoA deployment, follow these instructions: [https://fieldguide.automattic.com/developing-jetpack/jetpack-mu-wpcom/#woa-deployment]. A new version of `jetpack-mu-wpcom` will be released weekly, if you do not initiate a manual deployment.
\ No newline at end of file
+3. To initate a WoA deployment, follow these instructions: [https://fieldguide.automattic.com/developing-jetpack/jetpack-mu-wpcom/#woa-deployment]. A new version of `jetpack-mu-wpcom` will be released weekly, if you do not initiate a manual deployment.
+
+### Where to track new Verbum Issues
+
+If you stumble upon any issues or have any suggestions for possible changes to Verbum, you can find the relevant project board here: https://github.com/Automattic/jetpack/projects/45.
```

Comment:
I think an organization-wide board would work better here (see my comment here: https://github.com/Automattic/jetpack/issues/35314#issuecomment-1916315650)

```suggestion
If you stumble upon any issues or have any suggestions for possible changes to Verbum, you can find the relevant project board here: https://github.com/orgs/Automattic/projects/908/views/1
```

Reply:
Thank you @jeherve, I initially created one in Automattic then wrongly switched to this one 👍